src/Models/Rate.php

```diff
@@ -96,7 +96,7 @@ public function updater()
 
     public function rooms()
     {
-        return $this->hasMany(app('room'));
+        return $this->hasMany(app('room'), 'escaperoom_rate_id');
```

Comment:
This should be assigned to room, not rate

Reply:
Nice catch. Thank you 👯